cmd/kubectl-gadget/utils/trace.go

```diff
@@ -547,6 +556,18 @@ func waitForTraceState(traceID string, expectedState string) (*gadgetv1alpha1.Tr
 	return &returnedTraces, nil
 }
 
+// waitForTraceState waits for traces whom ID is given as parameter to be in the
+// expected state.
```

Comment:
```suggestion
// waitForTraceState waits for the traces with the ID received as parameter to be in the expected state.
```

Reply:
I try to cut a comment line when it exceeds 80 characters, so I prefer to stick with this here to remain consistent with the others comments.